src/Stan.hs

```diff
@@ -113,3 +114,12 @@ runTomlToCli TomlToCliArgs{..} = do
         fiasco -> do
             errorMessage "Could not get Configurations:"
             putTextLn $ prettyPrintTrial fiasco
+
+runCliToToml :: CliToTomlArgs -> IO ()
+runCliToToml CliToTomlArgs{..} = do
+    let toml = Toml.encode configCodec cliToTomlArgsConfig
+    case cliToTomlArgsFilePath of
+        Nothing   -> putTextLn toml
```

Comment:
Let's add an info message, something like, copy-paste this into `.stan.toml` and Stan will pick up this file on the next run. Or maybe say about any file and remind about --config-file option? :thinking: